Tallies/TallyClerks/imcWeightClerk_class.f90

```diff
@@ -0,0 +1,310 @@
+module imcWeightClerk_class
```

Comment:
I think this clerk might be more specific than it needs to be. Essentially, there's nothing necessarily IMC about it. Furthermore, rather than knowing anything about the material menu, you could just feed it a material map, I think? Maybe we don't have that in this branch. @valeriaRaffuzzi do you have or did you do something like this?

Reply:
I've been looking into this a bit and as far as I can tell the purpose of the collision estimator clerk is to add to the tally at the start of every collision, whereas we want to tally after a collision only if the particle is absorbed (reportHist instead of reportInColl). So it potentially does need a new clerk? Although I suppose that I can a lot more general, and use the weight response function and a material map.